src/components/DetailedPodcast/DetailedPodcast.stories.js

```diff
@@ -0,0 +1,12 @@
+
+import DetailedPodcast from './DetailedPodcast'
+export default {
+    title: 'DetailedPodcast',
+    component: DetailedPodcast
+}
+  
+const Template = args => <DetailedPodcast {...args} />
+  
+export const Default = Template.bind({})
+Default.args = {}
+      
```

Comment:
the Podcast is missing here aswell, but I can't find a fallback in case a podcast is missing - Is the Storybook working properly?

Reply:
No unfortunately storybook is not working for me. I will try to fix it at some point but at the moment i can design without it.